clients/web/src/views/body/SystemConfigurationView.js

```diff
@@ -65,7 +65,10 @@ var SystemConfigurationView = View.extend({
                 this.$('#g-settings-error-message').text(resp.responseJSON.message);
             }, this));
         },
-        'click #g-restart-server': restartServerPrompt
+        'click #g-restart-server': restartServerPrompt,
+        'click #g-core-banner-default-color': function () {
+            this.$('#g-core-banner-color').attr('value', this.defaults['core.banner_color']);
```

Comment:
Please check out [this helpful post](http://lucybain.com/blog/2014/attribute-vs-property/) on the distinction between an attribute and a property. If you needed to modify one of these directly, you'd want to use the [jQuery `.prop` method](http://api.jquery.com/prop/) method, since you'd be modifying a property of the DOM element, as opposed to an attribute of the HTML.

However, for `input` elements, jQuery helpfully provides [a `.val` mutator method](http://api.jquery.com/val/#val2). You should use that.